### PR TITLE
Update Dependencies and Time Display

### PR DESCRIPTION
**Pull request by Clashsoft**
* Updated dependencies to fix a vite problem.
* Fixed a time display bug (am/pm) and the placement of the note icon.
* Added an info text if the current time zone differs from the poll owner.

image
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,7 +114,5 @@
     "zone.js": "0.14.3"
   },
   "resolutions": {
```
Comment (Morphclue) [inline package.json:116]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -114,7 +114,5 @@
...
-    "vite@>=5.1.0 <=5.1.6": ">=5.1.7",
-    "undici@>=6.0.0 <=6.11.1": ">=6.11.1"
   }
```
Comment (Morphclue) [inline package.json:117]:
```suggestion
```

**Review by Morphclue — APPROVED** [review]
I just wanted to do the same thing, but you were faster than me. 

Just remove the empty resolutions and otherwise this looks good.

**Comment by Clashsoft** [comment]
I'd keep the resolutions as it is only a matter of time before we need them again.

**Pull request merged**